### DISCUSSION TITLE
Questions about: splits, goto line, grep search, ls diagnostics, and search within menus

### DISCUSSION THREAD

**Discussion by KnorrFG** [ki-editor/ki-editor#440] [Q&A] [answered]
Hey,
I've been toying around with ki and a couple of questions came up:

* How can I open a file in a new split?
* How can I jump to a line by number?
* How can I search through the content of all files in the repo? 
* The language server doesn't seem to work for me, how can I see the output log?
* How can I do text search in the help menu, that opens when I press space ? n in normal mode?

**Comment by wongjiahau** [comment, marked as answer]
> How can I open a file in a new split?

Unfortunately this is not implemented yet.

> How can I jump to a line by number?

For example if you want to jump to line 9:
`e 0 9 enter`

> How can I search through the content of all files in the repo?

I suppose you mean global search, which is `g /`.

> The language server doesn't seem to work for me, how can I see the output log?

The log path can be obtained at `ki log`.

> How can I do text search in the help menu, that opens when I press space ? n in normal mode?

First, press `esc` to enter Normal Mode, then you can treat the help panel like any other buffer, for example, use `/` to search, use `ctrl+u`/`ctrl+d` to scroll up/down.

**Reply by KnorrFG** [reply]
Thanks for the answers

**Reply by wongjiahau** [reply]
You’re welcome!
